platform-api/src/main/java/io/datacater/core/stream/KafkaStreamsAdmin.java

```diff
@@ -173,9 +173,14 @@ public Map<String, String> getMetadata() {
    * Inspect (or retrieve) the most recent events of the Topic.
    *
    * @param limit Number of records to retrieve.
+   * @param method Mode of operation for the retrieval of records. - Uniform: Distributed retrieval.
```

Comment:
Let's call it `sampleMethod` instead of `method` to align with the naming of the query parameter and be more explicit in general?